log.txt

```diff
@@ -0,0 +1,227 @@
+Checking CELERY_QUEUES matches Procfile
```

Comment:
this whole file should be deleted and not checked in.  probably also add this to your ``.gitignore_global``